Documentation/project-docs/linux-performance-tracing.md

```diff
@@ -89,7 +102,7 @@ Traces are best viewed using PerfView on Windows.  Note that we're currently loo
 2. Download PerfView from <http://aka.ms/perfview>.
 3. Run PerfView.exe
 
-	> ```bash
+	> ```
```

Comment:
Yup: ```` ```powershell ````, ```` ```bat ```` and ```` ```cmd ```` all are available

```powershell
# powershell

Add-AzureAccount
gcm git
```

```cmd
:: cmd

cd %UserProfile%
```

```bat
:: bat

cd %ProgramFiles(x86)%
```